test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -724,7 +725,10 @@ TEST(LiteInterpreterTest, BackPortByteCodeModelAllVersions) {
       x1 = torch.zeros(2, 2)
       x2 = torch.empty_like(torch.empty(2, 2))
       x3 = torch._convolution(input, self.weight, self.bias, [1, 1], [0, 0], [1, 1], False, [0, 0], 1, False, False, True, True)
-      return (x1, x2, x3)
+      x = 2 * torch.ones(1)
+      h = torch.ones(1)
+      torch.add(x, h, out=x)
```

Comment:
since you're expanding the model used to test all versions, maybe add a comment to this line since it's key to test v7->v6 backport